lib/main.dart

```diff
@@ -67,7 +67,7 @@ class ECellApp extends StatelessWidget {
           S.routeAboutUs: (_) =>
               BlocProvider(create: (_) => TeamCubit(FakeTeamRepository()), child: AboutUsScreen()),
         },
-        initialRoute: S.routeSplash,
+        initialRoute: S.routeLogin,
```

Comment:
Need for route Replacement?

Reply:
Sir ,I had changed it to bquiz route for convenience